Example/Showcase/Classes/DomainLayer/Chat/ChatService.swift

```diff
@@ -8,7 +8,7 @@ typealias Stream<T> = AnyPublisher<T, Never>
 final class ChatService {
 
     private var client: ChatClient = {
-        Chat.configure(crypto: DefaultCryptoProvider())
+        Chat.configure(derivator: DefaultDerivationProvider())
```

Comment:
I don't like the name derivator etc.
DefaultMnemonicDerivationProvider DefaultHDDerivationProvider

Reply:
What about `BIP44Provider`?

Reply:
sounds good to me!